MaterialLib/SolidModels/CreepBGRa.cpp

```diff
@@ -47,6 +49,18 @@ CreepBGRa<DisplacementDim>::integrateStress(
 
     const auto C = this->getElasticTensor(t, x, T);
     KelvinVector sigma_try = sigma_prev + C * (eps - eps_prev);
+
+    auto const& deviatoric_matrix = Invariants::deviatoric_projection;
+
+    KelvinVector s_try = deviatoric_matrix * sigma_try;
+    double const norm_s_try = Invariants::FrobeniusNorm(s_try);
```

Comment:
I'd make a one-liner, because s_try and deviatoric_matrix are not needed elsewhere, but it's more matter of taste...